package.json

```diff
@@ -6,8 +6,8 @@
   "author": "Daniel Kesler",
   "license": "MIT",
   "workspaces": [
-    "@celo/eslint",
-    "@celo/tsconfig"
+    "@celo/eslint-config",
+    "@celo/typescript"
   ],
   "devDependencies": {}
```

Comment:
Would it be worth adding/configuring the `packages-publisher` here?

Reply:
Yeah I think so. I was thinking of doing it when I tried to publish these two starting packages.